src/CLI/actioner/sonic_cli_vlan.py

```diff
@@ -29,6 +29,8 @@
 vlanDict = {}
 suppressVlanList = []
 
+api = cc.ApiClient()
```

Comment:
Question -- Do we need such changes in every module?? I see the session is maintained as global variable in cli_client. I feel individual actioners need not change their initialization sequence.

Reply:
There is no need for any change in individual actioners. This is only a minor optimization. "show vlan" internally called invoke_api() twice resulting in 2 instances of ApiClient. This just avoids that.